nsot/admin.py

```diff
@@ -1,15 +1,15 @@
 from __future__ import unicode_literals
 
-from custom_user.admin import EmailUserAdmin
 from django.contrib.auth import get_user_model
 from django.contrib import admin
 from django.utils.translation import ugettext_lazy as _
+from guardian.admin import GuardedModelAdmin
 
 from . import models
 
 
 # Register our custom User model
-class UserAdmin(EmailUserAdmin):
```

Comment:
It looks like `EmailUserAdmin` adds some stuff on top of regular `admin.ModelAdmin` so it may not be appropriate to just replace it with `GuardedModelAdmin`: https://github.com/jcugat/django-custom-user/blob/master/src/custom_user/admin.py#L10

Though if doing so doesn't mess with the admin stuff, then it's A-OK.

Reply:
That's a good catch. In fact, I don't think we need to change this base at all actually. I can't think of a single use-case at this time where we'll need to be applying object-level permissions to an individual user object.

Reply:
Makes sense. Reverted the change.